### PR TITLE
Split licence

### PR DESCRIPTION
**Pull request by whitfield-mj**

### CODE REVIEW

File: server/views/curfew/curfewAddressReviewForm.pug [hunk 1]
```diff
@@ -35,7 +35,7 @@ block content
                             label(for="hdcConsentNo") No
 
                         div#consentForm.panel.panel-border-narrow.js-hidden
-                            include includes/landlordConsent
+                            include ../licenceConditions/includes/landlordConsent
```
Comment (atodd-noms) [inline server/views/curfew/curfewAddressReviewForm.pug]: should this move into views/curfew ?

File: server/views/reporting/reportingInstructionsForm.pug [hunk 2]
```diff
@@ -20,18 +24,18 @@ block content
...
-                            input#town.form-control(name="address3" type="text" value=data.townOrCity)
+                            input#town.form-control(name="townOrCity" type="text" value=data.townOrCity)
 
                         div.pure-g
                             div.pure-u-1.pure-u-md-3-5
                                 div.form-group
                                     label.form-label(for="postcode") Postcode
-                                    input#postcode.form-control.small(name="postCode" type="text" value=data.postcode)
+                                    input#postcode.form-control.small(name="postcode" type="text" value=data.postcode)
                         div.pure-g
```
Comment (atodd-noms) [inline server/views/reporting/reportingInstructionsForm.pug]: In other places we've got postCode eg proposedAddress

Comment (whitfield-mj) [reply server/views/reporting/reportingInstructionsForm.pug]: Yeah I didn't know which to go with. Royal mail seem to have it as one word

File: server/views/curfew/curfewAddressReviewForm.pug (hunk 1 above)
Comment (whitfield-mj) [reply server/views/curfew/curfewAddressReviewForm.pug]: Correct

**Pull request merged**